app/mixins/Analytics.js

```diff
@@ -0,0 +1,11 @@
+var analytics = require("webmaker-analytics");
+
+// This mixin adds analytics event track capabilities to a component
+module.exports = {
+
+  event: function(action, options) {
```

Comment:
this should probably have a more indicative name, since it's going to be treated as a function tied to "whatever object loaded the mixin". If we're only using a single function in the mixin, we might in fact want to do this as a plain JS lib instead of a mixin, since `this.event` vs. `analytics.event` is the same level of complexity, but the second is scoped better.